packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -78,6 +92,28 @@
       padding: 0;
     }
   }
+
+  .#{$prefix}--card--inverse {
+    background-color: $inverse-02;
+
+    .#{$prefix}--card__title,
+    .#{$prefix}--card__content {
+      color: $inverse-01;
+    }
+
+    .#{$prefix}--card__eyebrow {
+      color: $text-03;
+    }
+
+    &:hover {
+      background-color: $inverse-hover-ui;
+    }
```

Comment:
Double check your `:hover` and make sure it's not being applied to the static card. I think I had a comment in a previous pull request suggesting to add a modifier for `link` type.

Reply:
got it